src/main/io/gps.c

```diff
@@ -82,6 +82,8 @@ float dTnav;             // Delta Time in milliseconds for navigation computatio
 int16_t nav_takeoff_bearing;
 navigationMode_e nav_mode = NAV_MODE_NONE;    // Navigation mode
 
+#define GPS_DISTANCE_FLOWN_MIN_GROUND_SPEED_THRESHOLD 15 // 5.4Km/h
```

Comment:
What unit is this?

Reply:
cm/s. I'll update the name to reflect that, thanks